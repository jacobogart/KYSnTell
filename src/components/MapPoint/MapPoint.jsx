import React from 'react'

export const MapPoint = () => {
  return (
    <div className='MapPoint'>
      <img src="https://www.shareicon.net/data/2016/09/15/829297_location_512x512.png" alt="Map Point"/>
    </div>
  )
}
