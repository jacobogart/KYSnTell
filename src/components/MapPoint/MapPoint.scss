.MapPoint {
  img {
    height: 50px;
  }
}