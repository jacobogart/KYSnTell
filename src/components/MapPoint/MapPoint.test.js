import React from 'react';
import { shallow } from 'enzyme';
import { MapPoint } from './MapPoint';

describe('MapPoint', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<MapPoint />)
    expect(wrapper).toMatchSnapshot();
  });
})