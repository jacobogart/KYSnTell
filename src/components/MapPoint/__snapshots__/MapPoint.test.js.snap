// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MapPoint should match snapshot 1`] = `
<div
  className="MapPoint"
>
  <img
    alt="Map Point"
    src="https://www.shareicon.net/data/2016/09/15/829297_location_512x512.png"
  />
</div>
`;
